src/Jackett.Common/Indexers/ShowRSS.cs

```diff
@@ -67,42 +67,39 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
             try
             {
                 xmlDoc.LoadXml(result.Content);
-                ReleaseInfo release;
-                string serie_title;
-
                 foreach (XmlNode node in xmlDoc.GetElementsByTagName("item"))
                 {
-                    release = new ReleaseInfo();
-
-                    release.MinimumRatio = 1;
-                    release.MinimumSeedTime = 172800; // 48 hours
-
-                    serie_title = node.SelectSingleNode(".//*[local-name()='raw_title']").InnerText;
-                    release.Title = serie_title;
-
-                    if ((query.ImdbID == null || !TorznabCaps.SupportsImdbMovieSearch) && !query.MatchQueryStringAND(release.Title))
+                    var serie_title = node.SelectSingleNode(".//*[local-name()='raw_title']").InnerText;
+                    if ((query.ImdbID == null || !TorznabCaps.SupportsImdbMovieSearch) &&
+                        !query.MatchQueryStringAND(serie_title))
                         continue;
 
-                    release.Comments = new Uri(node.SelectSingleNode("link").InnerText);
-
                     // Try to guess the category... I'm not proud of myself...
-                    var category = 5030;
-                    if (serie_title.Contains("720p"))
-                        category = 5040;
-                    release.Category = new List<int> { category };
+                    var category = serie_title.Contains("720p") ? TorznabCatType.TVHD.ID : TorznabCatType.TVSD.ID;
                     var test = node.SelectSingleNode("enclosure");
-                    release.Guid = new Uri(test.Attributes["url"].Value);
-                    release.PublishDate = DateTime.Parse(node.SelectSingleNode("pubDate").InnerText, CultureInfo.InvariantCulture);
-
-                    release.Description = node.SelectSingleNode("description").InnerText;
-                    release.InfoHash = node.SelectSingleNode("description").InnerText;
-                    release.Size = 0;
-                    release.Seeders = 1;
-                    release.Peers = 1;
-                    release.DownloadVolumeFactor = 0;
-                    release.UploadVolumeFactor = 1;
-                    release.MagnetUri = new Uri(node.SelectSingleNode("link").InnerText);
-                    releases.Add(release);
+                    var magnetUri = new Uri(node.SelectSingleNode("link").InnerText);
+                    var publishDate = DateTime.Parse(node.SelectSingleNode("pubDate").InnerText, CultureInfo.InvariantCulture);
+                    var infoHash = node.SelectSingleNode("description").InnerText;
+                    var guid = new Uri(test.Attributes["url"].Value);
```

Comment:
I think you can remove all the code related to guid and use the magnet instead

Reply:
Not sure I want to do that without testing it first to see why this particular value was used. I'll add TODO comment for future maintenance for now to bring attention to it.